release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2311,4 +2311,26 @@ link:https://access.redhat.com/solutions/5982961[{product-title} 4.7.8 container
 [id="ocp-4-7-8-upgrading"]
 ==== Upgrading
 
-To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions
\ No newline at end of file
+To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-7-9"]
+=== RHBA-2021:1365 - {product-title} 4.7.9 bug fix and security update
+
+Issued: 2021-05-03
+
+{product-title} release 4.7.9, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:1365[RHBA-2021:1365] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:1366[RHSA-2021:1366] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6001181[{product-title} 4.7.9 container image list]
+
+[id="ocp-4-7-8-bug-fixes"]
```

Comment:
*Drive-by comment* :car:  (was looking to see what the next 4.7 z-stream version was :smile: ).

Should be `[id="ocp-4-7-9-bug-fixes"]`